pkg/test/xds/cla_cache.go

```diff
@@ -15,7 +15,7 @@ type DummyCLACache struct {
 }
 
 func (d *DummyCLACache) GetCLA(ctx context.Context, meshName, meshHash string, cluster envoy_common.Cluster, apiVersion core_xds.APIVersion, endpointMap core_xds.EndpointMap) (proto.Message, error) {
-	return endpoints.CreateClusterLoadAssignment(cluster.Service(), d.OutboundTargets[cluster.Service()]), nil
+	return endpoints.CreateClusterLoadAssignment(cluster.Name(), d.OutboundTargets[cluster.Service()]), nil
```

Comment:
how is this related? was it wrong always?

Reply:
yes it was incorrect allways

Reply:
Also, it's `test/cla_cache.go` resource - not cla_cache change

Reply:
yeah I know, I'm just worried about this breaking something else

Reply:
If it breaks something we will see this in tests, but method `CreateClusterLoadAssignment` should get cluster name and in old version it got service name instead of cluster name